README.md

```diff
@@ -154,9 +154,9 @@ USAGE
   [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
 
 OPTIONS
-  -d, --outputdir=outputdir                                                         [default:
-                                                                                    ~/.commerce/force-app]
-                                                                                    Directory to output the conversion
+  -d, --outputdir=outputdir                                                         [default: ~/.commerc
+                                                                                    e/force-app] Directory to output the
+                                                                                    conversion
```

Comment:
nit: aren't they the same?

Reply:
It is the same, the readme generation sometimes get funky.